citi-onboarding/celeiro-dos-bambas
# Hotfix/change-footer-fields

## What I did

- I added a "**endereco**" field in footer-bar to backend.
- I remove the `node_modules/` folder in the root project.

## How to test

1. Open the terminal and go to the **server** folder of this project

1. In terminal:
   1. `git pull`
   1. `git checkout hotfix/change-footer-fiels`
   1. `yarn install`
   1. `yarn start`

1. In your browser go to [http://localhost:1337/admin](http://localhost:1337/admin)

1. Check the endereco field is present and configured

1. Test the things that you think are worth testing, even the ones that are not in this description
